src/input_context/events.rs

```diff
@@ -123,11 +123,8 @@ impl<A: InputAction> Copy for Ongoing<A> {}
 /// this event is triggered when the user releases the key.
 #[derive(Debug, Event)]
 pub struct Fired<A: InputAction> {
-    /// Action for which the event triggers.
-    pub marker: PhantomData<A>,
-
     /// Current action value.
-    pub value: ActionValue,
+    pub value: A::Output,
```

Comment:
There are other event structs in this file, I think we need to change them as well.

Reply:
Didn't notice those, fixed